### ISSUE TITLE
What is the number "44" in training script

### ISSUE THREAD

**Issue by yukitsuji** [JunaidCS032/MOTBeyondPixels#3]
In main_script_train.m (main_script_test.m), the number "44" is used for scaling the motion.
https://github.com/JunaidCS032/MOTBeyondPixels/blob/master/src/main_script_train.m#L167
I think "1.72" is the height of camera, but I don't know about "44".

Thanks!

**Comment by JunaidCS032** [comment]
This is a rough scaling factor that scales ego-vehicles motion so that it is in metric scale. This factor was computed using the Monocular ORBSLAM odometry (which we have provided in the Data folder) and the ground truth odometry provided by KITTI.
